modules/nw-enabling-multicast.adoc

```diff
@@ -16,8 +24,17 @@ You can enable multicast between Pods for your project.
 
 * Run the following command to enable multicast for a project:
 +
+[subs="attributes+"]
 ----
 $ oc annotate netnamespace <namespace> \ <1>
-    netnamespace.network.openshift.io/multicast-enabled=true
+    {annotation}
```

Comment:
It's `oc annotate netnamespace ...` for openshift-sdn, but `oc annotate namespace ...` for ovn-kubernetes ("namespace", not "netnamespace"). For both enable and disable. Otherwise this is correct.